src/App.js

```diff
@@ -1,45 +1,39 @@
-import React, { useState } from 'react';
+import React from 'react';
+import {
+  BrowserRouter as Router, Route, Link, Routes,
+} from 'react-router-dom';
 import Calculator from './components/Calculator';
 import HomePage from './components/HomePage';
 import FunnyQuot from './components/FunnyQuot';
 import './components/calculator.css';
 
```

Comment:
![image](https://github.com/jubaer919/math-magicians/assets/93039359/91c399d3-0759-4eb1-9d55-7c86e7e3e2da)
- [ ] It would be nice if you make your pull request summary more descriptive and professional by adding more details to it 👍